src/openms/source/ANALYSIS/MAPMATCHING/MapAlignmentAlgorithmKD.cpp

```diff
@@ -108,12 +108,17 @@ void MapAlignmentAlgorithmKD::fitLOWESS()
   for (Size i = 0; i < num_maps; ++i)
   {
     Size n = fit_data_[i].size();
+    const Param& lowess_param = param_.copy("LOWESS:", true);
     if (n < 50)
     {
       OPENMS_LOG_WARN << "Warning: Only " << n << " data points for LOWESS fit of map " << i << ". Consider adjusting RT or m/z tolerance or max_pairwise_log_fc, decreasing min_rel_cc_size, or increasing max_nr_conflicts." << endl;
+      TransformationModel::DataPoints identity = {{0,0}, {1,1}, {1e6,1e6}};
+      transformations_[i] = new TransformationModelLowess(identity, lowess_param);
```

Comment:
maybe use linear instead of lowess in that case?

Reply:
I think this is actually a quick workaround for a crash that occurred in the LOWESS algorithm.
But I agree that in that case it might make sense to just use a linear model.